example/json.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>flowchart.js · Playground</title>
+    <style type="text/css">
+        .end-element { background-color : #FFCCFF; }
+    </style>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/raphael/2.1.0/raphael-min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <!--<script src="http://flowchart.js.org/flowchart-latest.js"></script>-->
+   <script src="../release/flowchart.js"></script>
+    <script>
+
+        window.onload = function () {
+            var btn = document.getElementById("run"),
+                    cd = document.getElementById("code"),
```

Comment:
Since you're already putting these on their own lines just declare the "var" for each. Makes your intentions a little clearer.